packages/react/src/components/Loading/Loading-test.js

```diff
@@ -7,11 +7,51 @@
 
 import React from 'react';
 import Loading from '../Loading';
-import { mount } from 'enzyme';
+import { mount, shallow } from 'enzyme';
 import { settings } from 'carbon-components';
 
 const { prefix } = settings;
 
+describe('Loading - accessibility', () => {
+  describe('Automated Accessibility Testing', () => {
+    let wrapper;
+
+    beforeEach(() => {
+      wrapper = mount(<Loading id="test-id" />);
+      document.body.appendChild(wrapper.getDOMNode());
+    });
+
+    it('should have no Axe violations', async () => {
```

Comment:
How do you feel about separating out AVT1 tests like this? Should we just pick DAP? Would we expect this for all components? Would we ever need to interact with things and re-run DAP or aXe to get a complete scan?

Reply:
We chatted offline, but we can keep em separate for now and feel this out while adding tests to a few more components. Not sure if our users would be interested in one, the other, or both, but I think we should keep Axe and DAP for now.